orttraining/orttraining/training_api/utils.cc

```diff
@@ -57,7 +57,7 @@ bool GetParamNameFromGradient(const std::string& grad_name, std::string& param_n
   return false;
 }
 
-Status OrtValueLike(const SessionState& sess_state, const OrtValue& input_val, OrtValue& output_val) {
+Status OrtValueZeroLike(const SessionState& sess_state, const OrtValue& input_val, OrtValue& output_val) {
```

Comment:
To me AllocateOrtValueLike or OrtValueLike is more intuitive. Also please add a detailed description in the .h file explaining this function will allocate a new ortvalue like the input ortvalue with all zeroes.

Reply:
OK, changed to CreateOrtValueLikeAndFillWithZero to make the naming better self-explained.